test/models/order_test.rb

```diff
@@ -1,7 +1,46 @@
 require "test_helper"
 
 class OrderTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test "formatted date formats the date" do
+    order = Order.create
+
+    assert_equal Time.now.strftime("%B %e, %Y"), order.formatted_date
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.